content/project-news/reactos-0415-released.md

```diff
@@ -0,0 +1,73 @@
+---
+title:       "ReactOS 0.4.15 released"
+author:      "ReactOS Team"
+date:        2025-03-01
+tags:        [ "release" ]
+banner:      "img/project-news/reactos-0415-released/thumb.png"
+---
+
+We are pleased to announce the release of ReactOS 0.4.15! We have improvements to accessories and system tools including Notepad, Paint and the Input Method Editor, shell improvements, registry healing, USB boot fixes, audio fixes, memory management fixes, ISA bus driver fixes, and a new kernel debugger.
+
+This release is a culmination of the work of countless contributors since 0.4.14 was branched in 2020. This has been the largest release to date. There are nearly 8 times more commits this release than in 0.4.14. We are extremely proud of the progress we have made, and cannot wait to continue in this growth. Development on ReactOS has been growing exponentially! Let's dive in and see what's new.
+
+## Accessories and System Tools
+Katayama Hirofumi implemented quality of life improvements, performance enhancements, and new features such as the "text tool" in Paint and the "Now Printing" dialog in Notepad.
+
+Katayama has built out the Input Method Editor (or IME), which is a component that facilitates the typing of characters not originally present on the input devices by using a sequence of characters. This improves CJK support and allows installation of custom IMEs for different locales. Additionally, Japanese ReactOS recognizes and can use MZ-IME Japanese input.
+**Image here would be nice.**
+
+## Explorer Shell
+In 0.4.15 the Explorer shell was improved by Doug Lyons, Katayama Hirofumi MZ (katahiromz) and Carl Bialorucki (cbialorucki). They added the toolbar icon display, ZIP archive support, taskbar icons, and the addition of an "Internet Browser" icon on the desktop.
```

Comment:
On Zip Archive support, we might have to raise the name of @learn-more.

Reply:
Suggestion:@learn
```- ZIP archive support```
```+ ZIP archive support (courtesy of Mark Jansen)```